### PR TITLE
Database model and scripts for devfile

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Adds database model and SQL scripts for devfile

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13075
https://github.com/eclipse/che/issues/13079

### CODE REVIEW

**Comment by sleshchenko** [comment]
`devfileAsWorkspaceConfiguration` is outdated, could you please reset it to latest master and then rebase your branch against it? Or just create a draft/PR to master branch

File: wsmaster/integration-tests/postgresql-tck/src/test/java/PostgreSqlTckModule.java [hunk 1]
```diff
@@ -131,6 +139,17 @@ protected void configure() {
                 InstallerServerConfigImpl.class,
                 WorkspaceActivity.class,
                 VolumeImpl.class,
+                // devfile
+                ActionImpl.class,
```
Comment (sleshchenko) [inline wsmaster/integration-tests/postgresql-tck/src/test/java/PostgreSqlTckModule.java:143]: Please update `che/assembly/assembly-wsmaster-war/src/main/resources/META-INF/persistence.xml` with
```<class>org.eclipse.che.api.workspace.server.model.impl.devfile.DevfileImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.ProjectImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.ComponentImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.EntrypointImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.ActionImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.VolumeImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.SourceImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.EnvImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.CommandImpl</class>
        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.EndpointImpl</class>
```

**Review by sleshchenko — COMMENTED** [review]
Please add the needed test cases to Tck Workspace Test - > Workspace with Devfile is successful stored/searched by name

File: wsmaster/integration-tests/postgresql-tck/src/test/java/PostgreSqlTckModule.java (hunk 1 above)
Comment (mshaposhnik) [reply wsmaster/integration-tests/postgresql-tck/src/test/java/PostgreSqlTckModule.java:143]: Added

**Comment by mshaposhnik** [comment]
Rebased both branches up2date with parents;

**Comment by mshaposhnik** [comment]
ci-test

**Comment by sleshchenko** [comment]
Screenshot_20190410_141941
Looks like these changes are already in master and should not be present in this PR. Please remove them

**Pull request merged**